cli/cmd/component-delete.go

```diff
@@ -38,12 +38,14 @@ When run with no arguments, all components listed in the configuration are delet
 }
 
 var deleteNamespace bool
+var confirmDeleteComponent bool
```

Comment:
I'd call it simply `confirm` since we're already in the `component-delete.go` file

```suggestion
var confirm bool
```

Reply:
@iaguis it generates an error as confirm is already declared in `cluster-destroy.go`